### DISCUSSION TITLE
UniGetUI 3.1.0 alpha 1

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2275] [Show and tell]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.1.0-alpha1/total?style=for-the-badge&color=red)

<b>IMPORTANT:
This is the second development release to implement the UniGetUI rebranding.
Instaling this release will automatically migrate settings and other data to new folders (WingetUI -> UniGetUI), so rolling back to older versions of WingetUI may cause settings to be reset.
</b>

## Changes
 - Improvements to the rebranding
 - **lots** of bugfixes and improvements
 - Improvements to cache handling with GSudo
 - By popular demand, window position and size will be saved across app restarts
 - Change how information is rendered on the Package Details tab. Links should be copyable and should not show cut anymore

SHA256: `2B01939FC4E41B3CAF9D9E994B8E57DD3CEA016DFD03DA6CA8378819263EEF04`

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/WingetUI/releases/tag/3.1.0-alpha1'>UniGetUI 3.1.0 alpha 1</a>.</em>

**Comment by Spacellary** [comment]
Heya Marti! Just curious about the distribution of the software via package managers after the rebranding is rolled out, will there be new packages coexisting with the current WingetUI in Chocolatey, Scoop.sh, Winget...? Maybe just for a transition period? Or will the existing packages be updated to reflect the new version, so for example the Scoop `wingetui` [package](https://github.com/ScoopInstaller/Extras/blob/master/bucket/wingetui.json) will become `unigetui` directly?

**Reply by marticliment** [reply]
I am not the maintainer of the Scoop package, so I can't tell about it.

Regarding Scoop and Chocolatey, my plans were to mmigrate the package without having to install a new package

**Reply by Spacellary** [reply]
That'd be ideal.

**Comment by grimson73** [comment]
<img width="856" alt="image" src="https://github.com/marticliment/WingetUI/assets/45716059/fc7f0006-9eb1-4cb3-9999-1345afd9bd71">
Thanks for developing, chiming in this issue(?).
If I do a 'refresh' on Discover Packages the progress bar gets stuck in 'Loading..'. This happens with both UniGetUI versions. I think I recall I should get a full list offerings by the selected providers but I'm not sure.
Also if I deselect all providers then I get 'no packages were found' but selecting just WinGet or just Chocolately the progress bar never stops.

**Reply by marticliment** [reply]
Can you please attach your UniGetUI log? Please make sure to select "Log level: Debug" before exporting the log file

**Reply by grimson73** [reply]
Sure, thanks for replying! :)
What I did:
- Reset UniGetUI
- Changed Language to English instead of Locale
- Changed Package Managers just to Winget (+manual restart of UniGetUI to be sure)
- First thing after starting UniGetUI is then activating the refresh option in 'Discover Packages' and as expected the Loading... bar is in progress.
- With the progress bar running I went to 'more | UniGet Log | Adjusted to Log Level 5 w/reload log but no additions were observed.
- Export to a file
[UniGetUI Log.txt](https://github.com/user-attachments/files/15545704/UniGetUI.Log.txt)
- When going back to 'Discover Packages' the progress bar is still running and the state is 'Loading'.

It seems it doesn't log the discovery procedure?

**Reply by marticliment** [reply]
It should, but I think I have an idea of what is happening. I have opened https://github.com/marticliment/WingetUI/issues/2278 to track the issue

**Comment by rahaaatul** [comment]
Why not just UniGet?

**Comment by RobertoPuraVida** [comment]
Hello. I uninstalled WinGetUI (which was working fine) and installed the UnigetUI alpha1 version. It doesn't show my installed programs. The list It's always empty. What can I do? Any tipps?

**Reply by marticliment** [reply]
Reinstalling WinGetUI does show packages?

Please attach UniGetUI logs

**Reply by RobertoPuraVida** [reply]
> Reinstalling WinGetUI does show packages?
> 
> Please attach UniGetUI logs

WinGetUI works fine, all installed packages are shown. If I install UniGet
[UniGetUI-Log.txt](https://github.com/user-attachments/files/15593687/UniGetUI-Log.txt)
UI, it doesn't show my installed packages. That problem exists for me since alpha0 (and now with alpha1 as well).

**Reply by RobertoPuraVida** [reply]
And yes, if I reinstall WInGetUI, then it shows up all installed packages again. The problem is reproducable.

**Comment by bolens** [comment]
Chiming in on the infinitely loading packages-- I think it's a Windows or Winget issue.  I was having the same problems and tried just accessing winget via a terminal.  Running `winget upgrade` gave me an infinite load too, `Ctrl + C` to interrupt the loading gave me an error which led me [to the following MS forum post](https://answers.microsoft.com/en-us/windows/forum/all/error-the-group-or-resource-is-not-in-the-correct/4885b0c0-7483-4888-9ba8-1b6971d2c645).  Method 3 resolved it for me. The steps are as follows:

> Reset Windows Update Components
> 
> I would suggest you to manually reset Windows Update Components and then check for updates. Refer the steps mentioned below:
>     Type command prompt in the search bar on the Taskbar.
> 
>     Right click on the command prompt icon and select Run as administrator.
> 
>     Paste the following commands one by one and hit Enter:
> 
>     net stop wuauserv
> 
>     net stop cryptSvc
> 
>     net stop bits
> 
>     net stop msiserver
> 
>     ren C:\Windows\SoftwareDistribution SoftwareDistribution.old
> 
>     ren C:\Windows\System32\catroot2 Catroot2.old
> 
>     net start wuauserv
> 
>     net start cryptSvc
> 
>     net start bits
> 
>     net start msiserver
> 
> Restart the computer and check for updates.

**Reply by RobertoPuraVida** [reply]
Thank you for the tipp. Unfortunatelly it does not work. It's not a problem of Windows (or Winget), it's a problem of UniGetUI alpha. 
No sources are available. Guess this is the reason.
![Screenshot 2024-06-08 151927](https://github.com/marticliment/WingetUI/assets/171840004/c6ad077d-d134-498d-b80b-b1f753fbbb25)
